### ISSUE TITLE
Allow user to set excel workbook language

### ISSUE THREAD

**Issue by Olivia-Box-Power** [co-analysis/a11ytables#157]
The [analysis function guidance](https://analysisfunction.civilservice.gov.uk/policy-store/releasing-statistics-in-spreadsheets/#section-15) says that the excel document language should be set. It is not clear how to do this using openxlsx2. Its not clear whether these settings a specific to the xlsx file or whether they are an excel programme setting for the user.
